### PR TITLE
[test] 한정판매 서비스 - 인당 최대 주문 가능 수량 조회 기능 테스트코드 작성

### LINKED ISSUE #25: [test] 인당 최대 구매 가능 개수 조회 기능 테스트 코드 작성
The test only covers the happy path and a single error case (item not found). Consider adding tests for edge cases such as: empty item list (though validation should prevent it), duplicate item IDs in the request, and verifying the order of returned items matches the request order.

_Originally posted by @Copilot in https://github.com/speacial-T/Limito-limited-product/pull/24#discussion_r2589914863_

### PR DESCRIPTION
**Pull request by dobbi030**
#25 

### 변경사항

- 변경사항1

### 리뷰요청

- 리뷰요청1

### CODE REVIEW

File: src/test/java/com/limito/limitedproduct/application/service/LimitedProductServiceV1Test.java [hunk 1]
```diff
@@ -17,93 +17,71 @@
...
+	private void 예상한_예외가_발생했는지_검증한다() {
+		assertThatThrownBy(() -> limitedProductServiceV1.getPurchaseAmountLimits(wrongUUIDRequest))
 			.isInstanceOf(AppException.class)
 			.hasMessageContaining(LimitedProductErrorCode.PRODUCT_ITEM_NOT_FOUND.getMessage());
+		verify(limitedProductItemRepository, times(1)).findAllById(wrongUUIDItemIdList);
+	}
+
+	@Nested
+	@DisplayName("[성공] 인당 최대 구매 가능 수량 조회")
+	class LimtitedProductSuccessTest {
```
[the remark this answers is not in the document]
Comment (jjaenie) [reply src/test/java/com/limito/limitedproduct/application/service/LimitedProductServiceV1Test.java]: Lim"tit"ed 오타 있습니당

File: src/test/java/com/limito/limitedproduct/application/service/LimitedProductServiceV1Test.java [hunk 2]
```diff
@@ -1,109 +1,86 @@
...
+			상품_아이디가_주어지면_준비된_상품을_반환한다();
+			var response = 상품_아이디목록을_이용해서_상품을_조회한다();
+			상품_조회_결과를_검증한다(response);
+		}
+
+	}
+
+	@Nested
+	@DisplayName("[실패] 인당 최대 구매 가능 수량 조회")
+	class LimtitedProductFailTest {
```
Comment (jjaenie) [inline src/test/java/com/limito/limitedproduct/application/service/LimitedProductServiceV1Test.java]: 여기도 마찬가지로 오타입니당

File: src/test/java/com/limito/limitedproduct/application/service/LimitedProductServiceV1Test.java [hunk 3]
```diff
@@ -1,109 +1,86 @@
...
+	private void 예상한_예외가_발생했는지_검증한다() {
+		assertThatThrownBy(() -> limitedProductServiceV1.getPurchaseAmountLimits(wrongUUIDRequest))
 			.isInstanceOf(AppException.class)
 			.hasMessageContaining(LimitedProductErrorCode.PRODUCT_ITEM_NOT_FOUND.getMessage());
+		verify(limitedProductItemRepository, times(1)).findAllById(wrongUUIDItemIdList);
+	}
+
+	@Nested
+	@DisplayName("[성공] 인당 최대 구매 가능 수량 조회")
+	class LimtitedProductSuccessTest {
```
Comment (jjaenie) [inline src/test/java/com/limito/limitedproduct/application/service/LimitedProductServiceV1Test.java]: Lim"tit"ed 오타 있습니당
